### PR TITLE
Change resources path

### PR DESCRIPTION
**Pull request by minggo**
Now the resources path work ok on all platforms of cocos2d-x. 
I think cocos2d-iphone and cocos2d-html5 should do little modifications.

I think cocos2d-html5 should only add the variable definitions in WatermelonWithMe/resources-html5.js.

<pre>
var s_MainMenu = "MainMenu.ccbi";
var s_About = "About.ccbi";
</pre>

About cocos2d-iphone, i think it should only modify corresponding xcode projects settings.

### CODE REVIEW

**Comment by minggo** [comment]
The modification of coins_animation.plist make crash on cocos2d-html5. I will update the PR to resolve it. Please don't merge it.

**Comment by minggo** [comment]
I think cocos2d-js-html5 can run by merging this PR.

About cocos2d-iphone, as i said, should modify xcode project settings(re-add resources).
So we need your help, Riq.

**Comment by ricardoquesada** [comment]
Hi Minggo, sure, I will update the cocos2d-iphone project.

File: tests/tests-boot-jsb.js [hunk 1]
```diff
@@ -39,7 +39,7 @@ var tests_files = [
 		'UnitTest/UnitTest.js'
 	];
 for( var i=0; i < tests_files.length; i++) {
-	var name = "tests/" + tests_files[i];
+	var name = "" + tests_files[i];
 	cc.log(" Loading: " + name );
 	require( name );
 }
```
Comment (ricardoquesada) [inline tests/tests-boot-jsb.js:45]: @minggo 
for cocos2d-iphone, I am using the "tests" prefix for all the tests.
Could you do the same for cocos2d-x + Android ?

Comment (minggo) [inline tests/tests-boot-jsb.js:45]: There is a problem when using the "tests" prefix:
- tests/res/animations/animation-2.plist uses "res/animations/grossini.plist" as a path of spritesheet.

**Pull request merged**